doc/project/geidco.rst

```diff
@@ -1,7 +1,51 @@
-GEIDCO
-******
+GEIDCO V
+********
```

Comment:
Looking at the preview build of the docs page, [here](https://iiasa-energy-program-message-ix--312.com.readthedocs.build/projects/models/en/312/project/geidco.html): both "GEIDCO V" and "GEIDCO IV" show up in the left-hand nav pane/ToC.

This occurs because the "****" heading level is used in this document twice.

So I would suggest instead to use the next-lower heading level to distinguish the two project phases:
```rst
GEIDCO
******

…

GEIDCO V
========

…

GEIDCO IV
=========

…

Introduction
------------

…
```